idea-plugin/src/main/kotlin/io/github/composegears/valkyrie/ui/foundation/platform/BrowserUtil.kt

```diff
@@ -0,0 +1,25 @@
+package io.github.composegears.valkyrie.ui.foundation.platform
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.platform.LocalInspectionMode
+
+@Composable
+fun rememberBrowserUtil(): BrowserUtil {
+    if (LocalInspectionMode.current) return NoOpBrowserUtil
+
+    return BrowserUtilImpl
+}
+
+interface BrowserUtil {
```

Comment:
```suggestion
interface Browser {
```

Prefer not to use util in Compose.